warehouse/manage/views.py

```diff
@@ -1165,6 +1167,8 @@ def add_github_oidc_provider(self):
         if not self.oidc_enabled:
             raise HTTPNotFound
 
+        self.metrics.increment("warehouse.oidc.add_github_provider.attempt")
```

Comment:
```suggestion
        self.metrics.increment("warehouse.oidc.add_provider.attempt", tags=["provider:github"])
```

Reply:
and similar for others.

Reply:
Good call -- there's a few of these, so I'll roll them up.

Reply:
c945ee7d